src/main/java/com/labsynch/labseer/service/LabelSequenceServiceImpl.java

```diff
@@ -30,11 +30,16 @@ public Collection<LabelSequence> saveLabelSequenceArray(
 		for (LabelSequence labelSequence : labelSequences) {
 			LabelSequence savedLabelSequence;
 			try {
+				logger.info("Looking for existing LabelSequence: '" + labelSequence.getThingTypeAndKind() + "' '"
+						+ labelSequence.getLabelTypeAndKind() + "'");
 				savedLabelSequence = LabelSequence
 						.findLabelSequencesByThingTypeAndKindEqualsAndLabelTypeAndKindEquals(
 								labelSequence.getThingTypeAndKind(), labelSequence.getLabelTypeAndKind())
 						.getSingleResult();
+				logger.info("Found existing LabelSequence: '" + savedLabelSequence.getThingTypeAndKind() + "'' "
```

Comment:
```suggestion
				logger.info("Found existing LabelSequence: '" + savedLabelSequence.getThingTypeAndKind() + "' '"
```

Reply:
Thanks. Ha, I fixed this in the logging above which had the same issue but didn't notice it here. Good eye.